### PR TITLE
Allow WEBP Uploading

### PR DESCRIPTION
**Pull request by XVicarious**
**Changes**

Allow WEBP images to be uploaded in imageuploader.

2/2 (jellyfin/jellyfin-apiclient-javascript#24)

### CODE REVIEW

**Comment by dkanada** [comment]
Maybe we should just allow adding any image file from the frontend and let the backend handle errors with unsupported formats. @Bond-009 I know we have quite a few formats listed in the skia project, what do you think?

**Comment by Bond-009** [comment]
@dkanada Sounds good to me

**Comment by dkanada** [comment]
@XVicarious what about using something like `file.type.contains("image")` instead to avoid other formats getting ignored in the future?

**Comment by XVicarious** [comment]
How does that work? I'm not too versed in the specific frameworks, but I assumed that the vanilla JavaScript `String.prototype.startsWith` would both work and be appropriate for this application.

File: src/components/imageuploader/imageuploader.template.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h2 style="margin:0;">${HeaderAddUpdateImage}</h2>
                 <div class="imageEditor-dropZone fieldDescription">
                     <div>${LabelDropImageHere}</div>
                     <output id="imageOutput" class="flex align-items-center justify-content-center" style="position: absolute;top:0;left:0;right:0;bottom:0;width:100%;"></output>
-                    <input type="file" accept="image/png,image/x-png,image/jpeg" id="uploadImage" name="uploadImage" style="position: absolute;top:0;left:0;right:0;bottom:0;width:100%;opacity:0;" />
+                    <input type="file" accept="image/png,image/x-png,image/jpeg,image/webp" id="uploadImage" name="uploadImage" style="position: absolute;top:0;left:0;right:0;bottom:0;width:100%;opacity:0;" />
```
Comment (dkanada) [inline src/components/imageuploader/imageuploader.template.html]:
```suggestion
                    <input type="file" accept="image/*" id="uploadImage" name="uploadImage" style="position: absolute;top:0;left:0;right:0;bottom:0;width:100%;opacity:0;" />
```

Comment (anthonylavado) [reply src/components/imageuploader/imageuploader.template.html]: @XVicarious are you able to commit the suggestion from @dkanada?

**Pull request merged**